### ISSUE TITLE
Some type of daily quest reporting / notifications. 

### ISSUE THREAD

**Issue by Airalyn** [Googer/Professor-Pine-old#11]
Could be something as simple as `!quest String of text describing the quest` which then gets curated and users can request to see the current quest list for a region. (I don't think we want to have to maintain a list of pokestop locations or active potential quests since they change pretty frequently.)

**Comment by Googer** [comment]
Yeah I was thinking free-form text for the tasks themselves for this; we _may_ want the stops themselves though in Pine's data in an analogous setup to the gym data, though possibly without the geocoding part of it as even just for initial setup, that's *a lot* of geocoding requests (though that would be a one-time hit we may be willing to swallow - subsequent geocoding updates for new stops, updates, and deletions wouldn't be bad).

**Comment by cngodles** [comment]
I'm currently in the middle of developing a research tracking app for Pittsburgh, which we allow users to add pokestops/gyms. We have almost 500 in a database now, each with Lat/Lng data. If you were interested in trying to tie this all together, maybe I could see a daily pinned message in a regional channel with a list of research as it generates, or maybe they could call it with a command.

I'm not sure how to match pokestop text with the data, people don't enter it exact each time. However, I likely could match research or pokestops within a defined geofence.

It's currently a private beta and we've only had 20 unique users log in, and research ranges from 1 to 20 submissions per day, with many regions not covered.

**Comment by cngodles** [comment]
As stated in Discord yesterday. Here is a temp endpoint for pokestops/gyms:
https://eggs.15656.com/poke-research/pgp.php

And here is one for research:
https://eggs.15656.com/poke-research/pgp-r.php

If we launch this, I'll add a keyphrase to POST to the endpoint, but for now these are just open for testing use.
